src/allotropy/parsers/example_weyland_yutani/example_weyland_yutani_structure.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Optional
+
+import pandas as pd
+
+from allotropy.allotrope.allotrope import AllotropeConversionError
+from allotropy.parsers.lines_reader import CsvReader
+
+EMPTY_CSV_LINE = r"^[,\s]*$"
```

Comment:
I think this needs to be `^,*$`. We don't want lines like ",  ," to read as empty.

Reply:
Good catch - changed in the latest push.